task/06-conditions-n-loops-tasks.js

```diff
@@ -270,10 +299,8 @@ function isCreditCardNumber(ccn) {
  *   165536 (1+6+5+5+3+6 = 26,  2+6 = 8) => 8
  */
 function getDigitalRoot(num) {
-    throw new Error('Not implemented');
+  return num % 9;
```

Comment:
Wow. Does it really work? How did you find this solution?

Reply:
You didn't answer

Reply:
Two sources where I found a solution: